lib/mongoose.module.ts

```diff
@@ -12,13 +12,10 @@ import {
 
 @Module({})
 export class MongooseModule {
-  static forRoot(
-    uri: string,
-    options: MongooseModuleOptions = {},
-  ): DynamicModule {
+  static forRoot(options: MongooseModuleOptions): DynamicModule {
```

Comment:
This is a breaking change, could we support both cases in this module?

Reply:
We could, technically. Yes.

Although I'm more in favour to add a single breaking change & document it properly. Adaptation for consumers for this breaking change should be simple enough.

But either way: I'm not really willing to invest more time into this PR and make further changes, since it seems like to me that this PR will not get merged (open for more than 1 year)...